syntax = "proto3";
import "google/protobuf/struct.proto";

//protocol buffer compiler starting command
//python3 -m grpc_tools.protoc -I. --python_out=. --grpc_python_out=. t_service.proto

package t_service;

service t_srv {
	rpc get_doc (get_doc_request) returns (get_doc_response) {}
}

message get_doc_request {
  string document_name = 1;
  optional bytes str_params = 2;
  optional google.protobuf.Struct params = 3;
}

message get_doc_response {
  string document = 1;
}